md/release-notes.md

```diff
@@ -45,6 +45,18 @@ More info on how to use it [here](variables.md).
 
 EJB communication protocol with the engine is now deprecated. Removal is planned for 7.10 version.
 
+### Wildfly Bundle
+
+The Wildfly bundle has been deprecated in Bonita 7.9.
+
+The Wildfly bundle is a legacy solution to a compatibility problem between the Bonita Platform and the SQLServer database;
+the Bitronix JTA transaction manager used in the Tomcat bundle caused database deadlocks.
+With the replacement of the JTA transaction manager (cf. below), it's not an issue anymore.
+
+Therefore the Wildfly bundle has no more reason to be maintained. It's removal is planned for a future release.
```

Comment:
I would not enter in detailed explanations about why we had the WildFly bundle.
Please just state that it is deprecated.

Reply:
Or just state that if wildfly was used because the dB vendor is mssql 
switch to tomcat is ok now